cli/app/app.go

```diff
@@ -1202,80 +1209,60 @@ func print(name, trailing string, data []byte, toStdout, generateYaml bool, f *o
 
 // Up brings up rc, svc.
 func Up(c *cli.Context) {
-	//factory := cmdutil.NewFactory(nil)
-	//clientConfig, err := factory.ClientConfig()
-	//if err != nil {
-	//	logrus.Fatalf("Failed to get Kubernetes client config: %v", err)
-	//}
-	//client := client.NewOrDie(clientConfig)
-	//
-	//files, err := ioutil.ReadDir(".")
-	//if err != nil {
-	//	logrus.Fatalf("Failed to load rc, svc manifest files: %s\n", err)
-	//}
-	//
-	//// submit svc first
-	//sc := &api.Service{}
-	//for _, file := range files {
-	//	if strings.Contains(file.Name(), "svc") {
-	//		datasvc, err := ioutil.ReadFile(file.Name())
-	//
-	//		if err != nil {
-	//			logrus.Fatalf("Failed to load %s: %s\n", file.Name(), err)
-	//		}
-	//
-	//		if strings.Contains(file.Name(), "json") {
-	//			err := json.Unmarshal(datasvc, &sc)
-	//			if err != nil {
-	//				logrus.Fatalf("Failed to unmarshal file %s to svc object: %s\n", file.Name(), err)
-	//			}
-	//		}
-	//		if strings.Contains(file.Name(), "yaml") {
-	//			err := yaml.Unmarshal(datasvc, &sc)
-	//			if err != nil {
-	//				logrus.Fatalf("Failed to unmarshal file %s to svc object: %s\n", file.Name(), err)
-	//			}
-	//		}
-	//		// submit sc to k8s
-	//		scCreated, err := client.Services(api.NamespaceDefault).Create(sc)
-	//		if err != nil {
-	//			fmt.Println(err)
-	//		}
-	//		logrus.Debugf("%s\n", scCreated)
-	//	}
-	//}
-	//
-	//// then submit rc
-	//rc := &api.ReplicationController{}
-	//for _, file := range files {
-	//	if strings.Contains(file.Name(), "rc") {
-	//		datarc, err := ioutil.ReadFile(file.Name())
-	//
-	//		if err != nil {
-	//			logrus.Fatalf("Failed to load %s: %s\n", file.Name(), err)
-	//		}
-	//
-	//		if strings.Contains(file.Name(), "json") {
-	//			err := json.Unmarshal(datarc, &rc)
-	//			if err != nil {
-	//				logrus.Fatalf("Failed to unmarshal file %s to rc object: %s\n", file.Name(), err)
-	//			}
-	//		}
-	//		if strings.Contains(file.Name(), "yaml") {
-	//			err := yaml.Unmarshal(datarc, &rc)
-	//			if err != nil {
-	//				logrus.Fatalf("Failed to unmarshal file %s to rc object: %s\n", file.Name(), err)
-	//			}
-	//		}
-	//		// submit rc to k8s
-	//		rcCreated, err := client.ReplicationControllers(api.NamespaceDefault).Create(rc)
-	//		if err != nil {
-	//			fmt.Println(err)
-	//		}
-	//		logrus.Debugf("%s\n", rcCreated)
-	//	}
-	//}
+	logrus.Infof("We are going to create deployment controller and service for your dockerized application. \n" +
```

Comment:
Change all "Deployment controller" to "Deployment" or "Kubernetes Deployment" (when we say deployment controller we usually mean the thing that's responsible for creating/syncing Deployments).